packages/ui/components/profile/ProfileCardTemplate.tsx

```diff
@@ -0,0 +1,40 @@
+import { useTranslation } from 'react-i18next'
+
+import { MembershipPill } from './MembershipPill'
+import { ProfileImage } from './ProfileImage'
+
+export interface ProfileCardTemplateProps {
+  children: any
+  imgUrl: string
+  walletName: string
+  dateEstablished: Date
+  isMember: boolean
+  daoName: string
+}
+export const ProfileCardTemplate = ({
+  children,
+  imgUrl,
+  walletName,
+  dateEstablished,
+  isMember,
+  daoName,
+}: ProfileCardTemplateProps) => {
+  const { t } = useTranslation()
+
+  return (
+    <div className="flex flex-col justify-center items-center pt-10 rounded-lg border-2 border-border-primary">
+      <ProfileImage imgUrl={imgUrl} />
+      <div className="pt-6 title-text">{walletName}</div>
+      <div className="py-2 mb-4 font-mono caption-text">
+        {t('profile.est')}
+        {new Intl.DateTimeFormat('default', {
```

Comment:
@ezekiiel looked into adding this to i18n... but realized that's a js package. i don't think it makes sense to add ts to i18n package just for this. might make more sense to put this type of helper in utils once we can import it

Reply:
Yeah, agree that we should have a standard date formatter util